physical_properties/pKa/microstates/epik_microstates.csv

```diff
@@ -0,0 +1,21 @@
+OCCNc1ncnc(c12)oc(-c3ccc(cc3)OC)c2-c4ccc(cc4)OC 
```

Comment:
See comments on `oe_microstates.csv`; now it's becoming more clear and these are microstates out of OpenEye and Epik, but for which molecules?